README.md

```diff
@@ -67,6 +76,15 @@ Start the server
 
 `brew services start postgresql`
 
+
+Sometimes, the command brew services start postgresql may throw an error. You may then try to restart the server with brew services start postgresql. You can also restart the server by brew services stop postgresql followed by brew services start postgresql.
```

Comment:
replace by 'brew services restart postgresql' instead of stop followed by start.

you can use ``` to encompass commands, e.g. ```brew```

Reply:
changes commited :)